app/models/case_log.rb

```diff
@@ -156,7 +156,7 @@ def is_assured_shorthold_tenancy?
   end
 
   def is_internal_transfer?
-    !!(referral && referral.zero?)
+    !!(referral && referral == 1)
```

Comment:
This can just be `referral == 1`, the presence check is just there for 0 checks because `nil` doesn't respond to `.zero?`

Reply:
Nice! Will change that